plugins/modules/azure_rm_cdnprofile.py

```diff
@@ -129,7 +131,7 @@ def __init__(self):
             ),
             sku=dict(
                 type='str',
-                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft']
+                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft', 'standard_azurefrontdoor', 'premium_azurefrontdoor']
```

Comment:
line too long (196 > 160 characters)

Reply:
Appreciate the feedback!  Changed.  Curious if I missed where these type of requirements/coding standards are located?  Happy to conform!